app/assets/stylesheets/members/index.css.scss

```diff
@@ -41,6 +59,30 @@ $parallax-background-height: $parallax-window-height;
   }
 }
 
+// Mobile
+@include breakpoint($mobile) {
+  .parallax-static-content {
+    margin-top: 0;
+    padding-top: 200px;
+    padding-bottom: 300px;
+    height: 100px;
+    background-color: rgba(20, 20, 20, 0.7);
+    color: #9A9A8A;
+    position: relative;
+    z-index: 9;
+
+    h1 {
+      padding-bottom: 8px;
```

Comment:
Properties should be ordered background-color, color, font-family, font-size, font-weight, padding-bottom